python/ql/src/experimental/Security/CWE-089/ClickHouseSQLInjection.qhelp

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+If a database query (such as a SQL or NoSQL query) is built from
+user-provided data without sufficient sanitization, a user
+may be able to run malicious database queries.
+</p>
+</overview>
+
+<recommendation>
+<p>
+Most database connector libraries offer a way of safely
+embedding untrusted data into a query by means of query parameters
+or prepared statements.
+</p>
+</recommendation>
+
+<example>
+<p>
+In the following snippet, a user is fetched from the ClickHouse database
+using five different queries. In bad patterns query is build by formatting
+string with user-controlled data. In a qood pattern user-supplied parameter
+is send through a second argument (params) which is dict. Following cases
+show different types of communication with ClickHouse database.
+</p>
+
+<p>
+In the first case, the query executed via aioch Client. aioch - is a module
```

Comment:
As far as I know query help files should not contain too many examples.
I think one/two "BAD" cases (maybe one `aioch` case and one `Client class case) and one "GOOD" case should be enough.